dydx/dydxStateManager/dydxStateManager/AbacusStateManager.swift

```diff
@@ -139,23 +139,23 @@ public final class AbacusStateManager: NSObject {
         let appConfigsV2: AppConfigsV2
         if dydxBoolFeatureFlag.force_mainnet.isEnabled {
             deployment = "MAINNET"
-            appConfigsV2 = AppConfigsV2.companion.forApp
+            appConfigsV2 = AppConfigsV2.companion.forAppWithIsolatedMargins
```

Comment:
Nit: can move this out of "if" since we use forAppWithIsolatedMargins for all cases

Reply:
will address next PR